app/_src/policies/meshretry.md

```diff
@@ -11,12 +11,38 @@ This policy enables {{site.mesh_product_name}} to know how to behave if there is
 
 ## TargetRef support matrix
 
-| TargetRef type    | top level | to  | from |
-| ----------------- | --------- | --- | ---- |
-| Mesh              | ✅        | ✅  | ❌   |
-| MeshSubset        | ✅        | ❌  | ❌   |
-| MeshService       | ✅        | ✅  | ❌   |
-| MeshServiceSubset | ✅        | ❌  | ❌   |
+{% if_version gte:2.4.x %}
+{% tabs targetRef useUrlFragment=false %}
```

Comment:
do we have to do an if on the whole table? can we do this on a specific row?

Reply:
Yeah unfortunately not in general because the `to` column can be different